models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -0,0 +1,107 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_ldm_20230507.json",
+    "version": "1.0.0",
+    "changelog": {
+        "1.0.0": "Initial release"
+    },
+    "monai_version": "1.2.0rc5",
+    "pytorch_version": "1.13.1",
+    "numpy_version": "1.22.2",
+    "optional_packages_version": {
+        "nibabel": "5.1.0",
+        "lpips": "0.1.4"
+    },
+    "name": "BraTS MRI image latent diffusion generation",
+    "task": "BraTS MRI image synthesis",
+    "description": "A generative model for creating 3D brain MRI from Gaussian noise based on BraTS dataset",
+    "authors": "MONAI team",
+    "copyright": "Copyright (c) MONAI Consortium",
+    "data_source": "http://medicaldecathlon.com/",
+    "data_type": "nibabel",
+    "image_classes": "Flair brain MRI with 1.1x1.1x1.1 mm voxel size",
+    "eval_metrics": {},
+    "intended_use": "This is a research tool/prototype and not to be used clinically",
+    "references": [],
+    "autoencoder_data_format": {
+        "inputs": {
+            "image": {
```

Comment:
I think you also need to add other inputs. The network requires not only image, but also other args in forward function

Reply:
Yes! Thank you! I added timestep